### PR TITLE
Add new concourse chart

### PR DESCRIPTION
**Pull request by jknipper**
This PR adds the new concourse umbrella chart.

### CODE REVIEW

File: global/concourse/Chart.yaml [hunk 1]
```diff
@@ -1,16 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```
Comment (databus23) [inline global/concourse/Chart.yaml]: Add a proper description here

File: global/concourse/README.md [hunk 2]
```diff
@@ -1,246 +0,0 @@
-# Concourse Helm Chart
```
Comment (databus23) [inline global/concourse/README.md]: It would be nice to have a basic README explaining what this chart is how we did it.

Comment (databus23) [reply global/concourse/README.md]: Eat least we should briefly explain the following things:
* Daemonset per team with
* Logging reverse proxy

File: global/concourse/templates/daemonset.yaml [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+          value: {{ $.Values.concourse.concourse.worker.workDir | quote }}
+        - name: CONCOURSE_BIND_IP
+          value: "127.0.0.1"
+        - name: CONCOURSE_BIND_PORT
+          value: "7777"
+        - name: CONCOURSE_LOG_LEVEL
+          value: "info"
+        - name: CONCOURSE_TSA_HOST
+          value: "concourse-web:2222"
+        - name: CONCOURSE_TSA_PUBLIC_KEY
```
Comment (databus23) [inline global/concourse/templates/daemonset.yaml]: Maybe we should have seperate keys per team?

File: global/concourse/templates/daemonset.yaml [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+          value: "{{ $.Values.concourse.worker.keySecretsPath }}/host_key.pub"
+        - name: CONCOURSE_TSA_WORKER_PRIVATE_KEY
+          value: "{{ $.Values.concourse.worker.keySecretsPath }}/worker_key"
+        - name: CONCOURSE_GARDEN_BIND_PORT
+          value: "7777"
+        - name: CONCOURSE_BAGGAGECLAIM_DRIVER
+          value: "naive"
```
Comment (databus23) [inline global/concourse/templates/daemonset.yaml]: This is pretty bad performance wise, we need to use the standard btrfs one. I guess that's a little tricky with the Kubernetes volumes but this won't scale out all. Ideally the volume should be formatted with btrfs.

File: global/concourse/templates/daemonset.yaml [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+            port: 8888
+        lifecycle:
+          preStop:
+            exec:
+              command:
+              - "kill"
+              - "-s"
+              - "SIGUSR2"
+              - "1"
+        env:
```
Comment (databus23) [inline global/concourse/templates/daemonset.yaml]: We should add the following setting, ideally configureable by value:
```
      CONCOURSE_AUTH_DURATION:                 72h
      CONCOURSE_DEFAULT_BUILD_LOGS_TO_RETAIN:  50
      CONCOURSE_MAX_BUILD_LOGS_TO_RETAIN:      100
```

Comment (jknipper) [reply global/concourse/templates/daemonset.yaml]: This is set via values in official chart web daemonset.

File: global/concourse/templates/daemonset.yaml (hunk 3 above)
Comment (jknipper) [reply global/concourse/templates/daemonset.yaml]: We now have ssh keys per team for the workers.

**Comment by jknipper** [comment]
@databus23 Please have a look. We now have:
* volume provisioning with btrfs on worker spin-up
* ssh keys per team/worker

**Pull request merged**